src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -1,30 +1,33 @@
+import { useEffect, useState } from "react";
+import SightingList from "./SightingList";
+import { getWhaleSightings, NewSighting } from "../../clients/apiClient";
 import "./WhaleSightingViewer.scss";
 
 export function WhaleSightingViewer() {
+
+	const [page, setPage] = useState(1);
+
+	const [sightingsList, setSightingsList] = useState<NewSighting[]>();
+
+    useEffect(() => {
+        getWhaleSightings()
+            .then(data => setSightingsList(data));
+      }, []);
+
+
+	if (!sightingsList) return <p>Waiting for data...</p>
+
 	return 	<>
 		<h2 className="whale-sighting-heading">Whale Sighting Viewer</h2>
 		<div className="whale-sighting-page">
 			<div className="whale-sighting-map-view-button">Switch to Map View</div>
 			<div className="whale-sighting-filters">Filters to go here</div>
 			<div className="whale-sighting-sort">Sort to go here</div>
-			<ul className="whale-sighting-posts">
-				<li className="whale-sighting-post">
-					Post1 goes here
-					<button>Like</button>
-				</li>
-				<li className="whale-sighting-post">
-					Post2 goes here
-					<button>Like</button>
-				</li>
-				<li className="whale-sighting-post">
-					Post3 goes here
-					<button>Like</button>
-				</li>
-				<li className="whale-sighting-post">
-					Post4 goes here
-					<button>Like</button>
-				</li>
-			</ul>
-		</div>	
+			{<SightingList pageNum = {page} sightingsList = {sightingsList}/>}
+			<div className="page-buttons">
+				{page > 1 ? <p className="prevlink" onClick={() => setPage(page - 1)}>Previous</p> :<></>}
+            	{(sightingsList.length > page * 12) ? <p className="nextlink" onClick={() => setPage(page + 1)}>Next</p> : <></>}
```

Comment:
This is nice! Just check the indentation for these as they should be indented the same amount

Reply:
same indentation in vsc
![image](https://user-images.githubusercontent.com/122781084/228487147-fdf2dfb0-fc0a-4c70-945b-c840f8ca3b4a.png)